test/test.js

```diff
@@ -45,6 +45,11 @@ describe('remoteForm', function () {
     document.querySelector('button[type=submit]').click()
   })
 
+  it('installs remoteForm on form reference', function (done) {
+    remoteForm(htmlForm, async () => done())
+    document.querySelector('button[type=submit]').click()
```

Comment:
This test is missing any assertions. 

Can you please take a look at the test above and add some assertions to insure that the expected behaviour is indeed being installed on the element.

Reply:
Okay thank you for the update. Given that this is testing that the form is installed on the passed reference, I think this test is more complete.